src/main/java/org/broadinstitute/hellbender/tools/BwaMemIndexImageCreator.java

```diff
@@ -19,25 +23,51 @@
 )
 public final class BwaMemIndexImageCreator extends CommandLineProgram {
 
+    public static final String USE_EXISTING_INDEX_FULL_NAME = "useExistingIndexFilesIfAvailable";
+    public static final String USE_EXISTING_INDEX_SHORT_NAME = "useIndexFiles";
+
     @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
-              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
-              doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
     private String referenceFastaLoc = null;
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
-              shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
-              doc = "Output reference index image file (ending in \".img\").",
-              optional = true)
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc = "Output reference index image file (ending in \".img\").",
+            optional = true)
     private String referenceIndexImageOutputLoc = null;
 
+    @Argument(fullName = USE_EXISTING_INDEX_FULL_NAME,
```

Comment:
As pointed by @tedsharpe, this option can lead to bugs if the indexes were created with previous or later versions, that are not coupled with the JNI. If you want to maintain this argument for convenience, I will add the barclay annotation `@Advance` to mark as a __"only use if you know what you are doing"__.